docker-standalone-ethereum-studio-on-ubuntu/README.md

```diff
@@ -0,0 +1,3 @@
+# Deployment of Ethereum Studio docker in standalone mode
+
+This template allows you to deploy an UbuntuServer 15.10 VM with Ethereum Studio docker in standalone mode.
```

Comment:
Can you add a little bit more documentation about what is Ethereum Studio, what it means to be standalone, if users need to do anything after this template executes, if there's any pre-setup required etc.